README.md

```diff
@@ -135,38 +135,33 @@ Or use parallel for faster tests. Note that these tests run in multiple VMs by c
-- **/test-centos-e2e-release-1-2** runs feature e2e tests with CAPM3 API version v1beta1 and branch release-1.2 on CentOS
+- **/test-ubuntu-e2e-feature-release-1-2** runs feature e2e tests with CAPM3 API version v1beta1 and branch release-1.2 on Ubuntu
+- **/test-centos-e2e-feature-release-1-2** runs feature e2e tests with CAPM3 API version v1beta1 and branch release-1.2 on CentOS
 
 Release-1.1 branch:
 
-- **/test-ubuntu-e2e-release-1-1** runs e2e tests with CAPM3 API version v1beta1 and branch release-1.1 on Ubuntu
-- **/test-centos-e2e-release-1-1** runs e2e tests with CAPM3 API version v1beta1 and branch release-1.1 on CentOS
+- **/test-ubuntu-e2e-feature-release-1-1** runs e2e tests with CAPM3 API version v1beta1 and branch release-1.1 on Ubuntu
+- **/test-centos-e2e-feature-release-1-1** runs e2e tests with CAPM3 API version v1beta1 and branch release-1.1 on CentOS
 
 ### Upgrade tests
 
-- **/test-upgrade-e2e-main** runs e2e upgrade tests from CAPM3 API version v1alpha5/branch release-0.5 to CAPM3 API version v1beta1/branch main on Ubuntu
+- **/test-e2e-upgrade-main** runs e2e upgrade tests from CAPM3 API version v1alpha5/branch release-0.5 to CAPM3 API version v1beta1/branch main on Ubuntu
 
 ### Keep VM
 
 After the e2e test is completed, Jenkins executes another script to clean up the environment first and then deletes the VM. However, sometimes it may be desirable to keep the VM for debugging purposes. To avoid clean up
-and deletion operations, there are separate triggers phrases as below:
-
-- **/keep-test-ubuntu-e2e-main** run keep e2e tests with CAPM3 API version v1beta1 and branch main on Ubuntu
-- **/keep-test-centos-e2e-main** run keep e2e tests with CAPM3 API version v1beta1 and branch main on CentOS
-- **/keep-test-ubuntu-e2e-release-1-3** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.3 on Ubuntu
-- **/keep-test-centos-e2e-release-1-3** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.3 on CentOS
-- **/keep-test-ubuntu-e2e-release-1-2** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.2 on Ubuntu
-- **/keep-test-centos-e2e-release-1-2** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.2 on CentOS
-- **/keep-test-ubuntu-e2e-release-1-1** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.1 on Ubuntu
-- **/keep-test-centos-e2e-release-1-1** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.1 on CentOS
-- **/keep-test-upgrade-e2e-main** runs keep e2e upgrade tests from CAPM3 API version v1alpha5/branch release-0.5 to CAPM3 API version v1beta1/branch main on Ubuntu
-
-Note: Test VM created with these phrases will not be kept forever but deleted after 24 hours.
+and deletion operations, use `keep-` prefix e.g:
+
+- **/keep-test-ubuntu-e2e-integration-main** run keep e2e tests with CAPM3 API version v1beta1 and branch main on Ubuntu
```

Comment:
Could we have `-keep` postfix instead of `keep-` prefix ?

Reply:
Yes we can except it requires huge changes as we need also to adapt ansible tests to avoid confusion ! do you think it worth it ?

Reply:
I have no idea how much work it is, so if you say its a lot of work, then we keep it this way, probably not worth it.

Reply:
To me it is relatively big changes and needs to cover Ansible tests I would prefer to have it in a separate PR and keep the current limited to e2e tests